### ISSUE TITLE
Transliteration of ukrainian language

### ISSUE THREAD

**Issue by dchaplinsky** [barseghyanartur/transliterate#7]
Hello.

There are standardized transliteration table for ukrainian language (approved by the Cabinet in 2005):
http://uk.wikipedia.org/wiki/Транслітерація

While it's a bit weird, it's used a lot in official stuff: international passports, visa/master cards, etc

I think your lib might benefit from using such a standardized version.

**Comment by sir-malefici** [comment]
This standart on official gov site: http://zakon2.rada.gov.ua/laws/show/55-2010-п

There is some transliteration specifics.

Some letters has different transliteration based on their position. This is an example for cyrillic letter "Я":

 * If it is the first letter in the word: **Я**готин -> **Ya**hotyn
 * If it is *not* the first letter in the word: Феодосі**я** -> Feodosi**ia**

Sequence of letters **зг** has next transliteration: **zgh**. So, it should be processed with more high priority than previous specific.

All this mean that we should use regular expressions. 

Also, if somebody knows, is there any language with same transliteration specifics?

**Comment by dchaplinsky** [comment]
@sir-malefici, you are right about regexes, this is why I've created https://github.com/dchaplinsky/translit-ua which now contains 19 different transliteration schemes for ukrainian and russian languages (including official one).

Speaking of transliteration specifics, russian language clearly has it:
https://ru.wikipedia.org/wiki/%D0%A2%D1%80%D0%B0%D0%BD%D1%81%D0%BB%D0%B8%D1%82%D0%B5%D1%80%D0%B0%D1%86%D0%B8%D1%8F_%D1%80%D1%83%D1%81%D1%81%D0%BA%D0%BE%D0%B3%D0%BE_%D0%B0%D0%BB%D1%84%D0%B0%D0%B2%D0%B8%D1%82%D0%B0_%D0%BB%D0%B0%D1%82%D0%B8%D0%BD%D0%B8%D1%86%D0%B5%D0%B9#.D0.A1.D1.80.D0.B0.D0.B2.D0.BD.D0.B8.D1.82.D0.B5.D0.BB.D1.8C.D0.BD.D0.B0.D1.8F_.D1.82.D0.B0.D0.B1.D0.BB.D0.B8.D1.86.D0.B0_.D1.81.D0.B8.D1.81.D1.82.D0.B5.D0.BC_.D1.82.D1.80.D0.B0.D0.BD.D1.81.D0.BB.D0.B8.D1.82.D0.B5.D1.80.D0.B0.D1.86.D0.B8.D0.B8
